docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -56,6 +56,36 @@ Different settings can be applied to a load balancer service in AWS using _annot
 
 The information for the annotations for AWS is taken from the comments on [aws.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go)
 
+# OpenStack
+This section describes all the possible configurations which can
+be used when using OpenStack with Kubernetes.
+
+## cloud.conf
+Kubernetes knows how to interact with OpenStack via the file cloud.conf.It is the file that will tell Kubernetes where are your credentials and OpenStack auth endpoint.
```

Comment:
Minor spelling nit - need a space between `config.conf.` and `It is...`

Reply:
Slightly better wording "will tell Kubernetes where are your credentials" -> "will provide Kubernetes with credentials and location for the OpenStack auth endpoint."